cmd/kueue/main.go

```diff
@@ -279,6 +280,7 @@ func setupControllers(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manag
 		jobframework.WithEnabledExternalFrameworks(cfg.Integrations.ExternalFrameworks),
 		jobframework.WithManagerName(constants.KueueName),
 		jobframework.WithLabelKeysToCopy(cfg.Integrations.LabelKeysToCopy),
+		jobframework.WithIntegrationOptions(appsv1.SchemeGroupVersion.WithKind("Deployment").String(), cfg.Integrations.DeploymentOptions),
```

Comment:
nit: keep it next to the other call to `.WithIntegrationOptions` a few lines above.